sympy/ntheory/modular.py

```diff
@@ -253,3 +258,44 @@ def combine(c1, c2):
         if symmetric:
             return symmetric_residue(n, m), m
         return n, m
+
+
+def crt_cartesian(rem, mod):
```

Comment:
A natural structure for this would be a dictionary whose keys are the moduli and whose values are the corresponding remainders: `{7:[3, 5], 11: [3,7]}`

Reply:
@mijo2 Please address this comment in your PR.

Reply:
Ok @namannimmo10 on it. Thanks!

Reply:
From the docstring of `crt`

> Note: the order of gf_crt's arguments is reversed relative to crt,
and that solve_congruence takes residue, modulus pairs.

so signatures are 

* `crt(mod, rem)`
* `gf_crt(rem, mod)`
* `solve_congruence(*rem_mod_pair)`

Also there is a O(n) check in `crt` that the result is consistent with the assumption that the moduli are pairwise coprime. Perhaps this should follow that example.